### PR TITLE
Update Hypershift docs for v0.6

### PR DESCRIPTION
**Pull request by Harkishen-Singh**
Signed-off-by: Harkishen-Singh <harkishensingh@hotmail.com>

# Description

This PR updates documentation related to Hypershift. On a high-level, the changes include:
1. Arguments are mandatory. Example: `clone`
2. Some configuration options are as flags for `clone`, the rest fall under the configuration file for clone.
3. Configuration field description were given a subheading level where they should have been as headings

# Links

Fixes #[insert issue link, if any]

# Writing help

For information about style and word usage, see the [style guide](https://docs.timescale.com/about/latest/contribute-to-docs/)

# Review checklists

Reviewers: use this section to ensure you have checked everything before approving this PR:

## Subject matter expert (SME) review checklist

*   [ ] Is the content technically accurate?
*   [ ] Is the content complete?
*   [ ] Is the content presented in a logical order?
*   [ ] Does the content use appropriate names for features and products?
*   [ ] Does the content provide relevant links to further information?

## Documentation team review checklist

*   [ ] Is the content free from typos?
*   [ ] Does the content use plain English?
*   [ ] Does the content contain clear sections for concepts, tasks, and references?
*   [ ] Have any images been uploaded to the correct location, and are resolvable?
*   [ ] If the page index was updated, are redirects required
      and have they been implemented?
*   [ ] Have you checked the built version of this content?

### CODE REVIEW

File: use-timescale/migration/about-hypershift.md [hunk 1]
```diff
@@ -67,19 +67,26 @@ URIs in the configuration file.
...
+
 |Short command|Long command|Example use|Description|
 |-|-|-|-|
-|-c|--config-file|-C <CONFIG_FILE>|Path for hypershift configuration file|
 |-s|--source|-s <SOURCE_URI>|URI to the source database|
 |-t|--target|-t <TARGET_URI>|URI to the target database|
 ||--source-password|--source-password|Password for the source database|
 ||--target-password|--target-password|Password for the target database|
-||--ignore-missing-time-index||Ignore that hypertables may be missing a time index|
+||--parallel|--parallel <NUM_WORKERS>|Number of workers to fetch data parallely from the target database|
```
Comment (Loquacity) [inline use-timescale/migration/about-hypershift.md:83]:
```suggestion
||--parallel|--parallel <NUMBER>|Number of workers that can simultaneously fetch data from the target database|
```

File: use-timescale/migration/about-hypershift.md [hunk 2]
```diff
@@ -67,19 +67,26 @@ URIs in the configuration file.
...
+||--verify|--verify|Verify the migrated data by comparing the source and target database|
+||--ignore-missing-time-index|--ignore-missing-time-index|Ignore that hypertables may be missing a time index|
+||--hide-progress|--hide-progress|Hide migration progress bar|
+||--verbose|--verbose|Display verbose logs with progress bar|
+|-c|--config-file|-c <CONFIG_FILE>|Path for advanced hypershift configuration file. For details, please see [hypershift-config]|
```
Comment (Loquacity) [inline use-timescale/migration/about-hypershift.md:88]:
```suggestion
|-c|--config-file|-c <PATH>|Path to an advanced hypershift configuration file. For more information, see [hypershift-config]|
```

File: use-timescale/migration/hypershift-config.md [hunk 3]
```diff
@@ -305,46 +302,53 @@ exclude_schemas: [<string>] # Default = []
...
-# Specify the logging format to be used.
-log_format: <VALID: json, text> # Default = text
+
 # Create the target database.
 create_db: <boolean> # Default = false
+
 # Drop constraints that prevent hypertables from being created.
 #
-# Some constraints like foreign keys pointing to a Hypertable cannot be
+# Some constraints like Foreign Keys pointing to a Hypertable cannot be
```
Comment (Loquacity) [inline use-timescale/migration/hypershift-config.md:338]:
```suggestion
# Some constraints like foreign keys that point to a hypertable cannot be
```

File: use-timescale/migration/about-hypershift.md [hunk 4]
```diff
@@ -67,19 +67,26 @@ URIs in the configuration file.
 To see all available configuration options for hypershift, use this command:
 
 ```bash
-docker run -ti timescale/hypershift:0.5 --help
+docker run -ti timescale/hypershift:0.6 --help
 ```
 
+Hypershift accepts operations it should perform as commands. These commands should be passed as arguments. Supported commands: `clone`
```
[the remark this answers is not in the document]
Comment (Loquacity) [reply use-timescale/migration/about-hypershift.md:73]:
```suggestion
You can give hypershift commands to perform other operations. 
These commands are passed as arguments. Currently, the only 
supported command is `clone`, which is used to FIXME.
```

I'm not sure if this is right, and it needs a little more exposition.

**Review by Loquacity — CHANGES_REQUESTED** [review]

File: use-timescale/migration/about-hypershift.md (hunk 4 above)
Comment (Harkishen-Singh) [reply use-timescale/migration/about-hypershift.md:73]: The `clone` command in Hypershift is basically an instruction to Hypershift to fork/make a full-copy of the contents of a source database into a target database. I think it was misunderstood.

Comment (Loquacity) [reply use-timescale/migration/about-hypershift.md]: @Harkishen-Singh Updated for you, let me know what you think.

**Comment by Loquacity** [comment]
Moved this to a new PR, because the period in the branch name is making the preview build fail.

**Pull request closed without merging**